src/main/java/io/github/eocqrs/kafka/consumer/KfConsumer.java

```diff
@@ -75,11 +73,7 @@ public KfConsumer(final ConsumerSettings<K, X> settings) {
    * @param settings XML settings
    */
   public KfConsumer(final XML settings) {
-    this(
```

Comment:
I like old formatting more

Reply:
@l3r8yJ let's integrate checkstyle in eo-kafka

Reply:
@h1alexbel what about formatting, do it in 1 line or not?

Reply:
@l3r8yJ also, it makes sense to do in one line what we can do in one line

Reply:
@l3r8yJ hear you, then it stays as it is